fontbe/src/features/marks.rs

```diff
@@ -80,6 +80,11 @@ impl MarkGroup<'_> {
             self.marks
                 .iter()
                 .map(|(name, _)| glyph_order.glyph_id(name).unwrap())
+                .chain(
```

Comment:
okay cool I think I'd just misunderstood the logic here, and thought that only the marks which were 'acting as marks' should go in the mark filtering set in this case.

Reply:
The idea is that you want the mark-to-mark attachment to apply to these bases and marks only and ignore any possible intervening mark. Since any GDEF-mark glyph not in the mark attachment set will be ignored, the base marks will be ignored as well and the lookup will not be applied. Mark filtering sets do not care about the rule the mark is taking in this lookup (base or mark) only its GDEF class.